smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/waiters/JMESPathVisitor.kt

```diff
@@ -54,15 +54,13 @@ class JMESPathVisitor(
     val writer: SwiftWriter,
```

Comment:
Add support for wildcard flattening expression (`[*]`) and the `keys()` function of JMESPath language.